app/src/main/java/edu/uw/myuw_android/NavDrawerMain.kt

```diff
@@ -62,6 +63,16 @@ class NavDrawerMain : AppCompatActivity() {
             drawerLayout.closeDrawer(navView)
         }
 
+        navView.setNavigationItemSelectedListener {
+            if (currentMenuItemId != it.itemId) {
+                navController.navigate(it.itemId)
+                navView.setCheckedItem(it.itemId)
+                currentMenuItemId = it.itemId
+            }
+            drawerLayout.closeDrawer(navView)
+            return@setNavigationItemSelectedListener false
```

Comment:
I looked up this return at labels, and I don't see a reason why we are using this since we are not in a loop. It can just simply be "false" to deactivate the listener.

Reply:
updated to false